src/utils/EmailType.tsx

```diff
@@ -0,0 +1,34 @@
+import { GenericIdentifierType } from './GenericIdentifierType';
+import { FunctionalComponent, h } from '@stencil/core';
+
+/**
+ * This class specifies a custom renderer for Email addresses.
+ * @extends GenericIdentifierType
+ */
+export class EmailType extends GenericIdentifierType {
+  getSettingsKey(): string {
+    return 'DateType';
+  }
+
+  hasCorrectFormat(): boolean {
+    return true;
+    //const regex = new RegExp('(\w|\(|\)|\.|\"|\(|\)|,|:|;|<|>|@|\[|\]|\\|!|#|\$|%|&|\'|\*|\+|-|/|\?|\^|`|{|\||}|~)+@((((\(.*\))?\w+(\(.*\))?\.)+(\(.*\))?\w+(\(.*\))?\w+)|(\[((IPv6:((\w+:+)+\w+))|(\d+\.)+\d+)\]))$');
+    //return regex.test(this.value);
+  }
+
+  init(): Promise<void> {
+    return;
+  }
+
+  isResolvable(): boolean {
+    return false;
+  }
+
+  renderPreview(): FunctionalComponent<any> {
+    return (
+      <a href={'mailto:' + this.value} target="_blank" class={'font-mono text-sm text-blue-400'}>
```

Comment:
The ORCiDType Renderer renders multiple emails as a comma separated list (e.g. 0000-0001-9842-9718). It would be nice if your renderer accepts this kind of input and renders it nicely. This is a suggestion:

![Untitled-2024-01-22-1408](https://github.com/kit-data-manager/pid-component/assets/43142805/9ebfdd74-8e6c-4f59-804c-27814a6bd2e8)

Keep in mind that the footprint should be minimal for optimal readability. Please make the comma recognition user-friendly. It shouldn't matter if there are spaces before and after the comma (e.g. `a,b` or `a , b` or  `a ,b` or `a, b`).

Reply:
I can include that. Can this also happen to other types of values, like URLs?

Reply:
Also accoding to wikipedia, commas can be part of mail adresses according to [wikipedia](https://en.wikipedia.org/wiki/Email_address#Local-part). In that case ill just narrow down the valid emails to what is more commonly expected by most mail services.